tools/image-builder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get -y install \
+    build-essential \
+    ccache \
+    clang \
+    cmake \
+    golang \
+    haskell-stack \
+    python2.7-dev \
+    wget \
+    zlib1g-dev \
+    zsh
+
+ADD build_boosts.zsh /root/
+
+RUN ["zsh", "/root/build_boosts.zsh"]
+
+VOLUME /root/.ccache /root/.stack /root/bond
```

Comment:
Feels weird that this comes after the `RUN`s. I can imagine that it's fine (because `RUN` registers a command, but doesn't actually execute it?), but still. It's weird. How about putting it between `FROM` and the first `RUN`?